### PR TITLE
Add instruction for influxdb backup and restore

### PR DESCRIPTION
**Pull request by emiliyank**

### CODE REVIEW

File: deployment/influx_backup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Backup influxdb
+In this document we describe how to perform backup and restore of influxdb v2 from docker container.
+
+1. Backup the data from a docker container:
+`docker exec -it <container_id> influx backup <backup_location> -t <secret_token>`
```
Comment (cupakob) [inline deployment/influx_backup.md:5]: the secret token can be retrived from the `.env`?

Comment (emiliyank) [reply deployment/influx_backup.md:5]: If needed yes but in this case the document describes how to perform the backup and restore manually from the server terminal. 
I think the idea is to setup cronjob with daily backup and we will use the restore only if needed. And maybe it is good idea to transfer the backup files on other location so we don't lose data if something happens with the whole server.

Comment (cupakob) [reply deployment/influx_backup.md:5]: thats okay, my question was only about the token, because i need to know where or how i have to retrive it.

Comment (emiliyank) [reply deployment/influx_backup.md:5]: you can get it from the .env - yes

**Review by cupakob — APPROVED** [review]

**Review by burberius — APPROVED** [review]

**Pull request merged**